srcalc/src/main/java/gov/va/med/srcalc/web/view/VariableView.java

```diff
@@ -38,7 +41,7 @@ public String getReferenceInfo()
     }
     
     /**
-     * @see VariableEntry.makeDynamicValuePath(String)
+     * @see VariableEntry#makeDynamicValuePath(String)
```

Comment:
Look at the documentation javadoc actually produces here: `@see` just makes a "see also" link at the bottom. In cases like this, I think English like `See {@link VariableEntry#makeDynamicValuePath(String)}.` is much friendlier.